apps/explorer/tests/utils/test_utils.py

```diff
@@ -0,0 +1,170 @@
+import pandas
+import yaml
+
+from apps.core import factories
+from apps.core.models import PixelSet
+from apps.core.tests import CoreFixturesTestCase
+
+from ...utils import (
+    PIXELSET_EXPORT_META_FILENAME, PIXELSET_EXPORT_PIXELS_FILENAME,
+    export_pixelsets
+)
+
+
+class ExportPixelsSetTestCase(CoreFixturesTestCase):
+
+    def _assert_archive_is_valid(self, zip_archive):
+
+        assert len(zip_archive.namelist()) == 2
+        assert PIXELSET_EXPORT_META_FILENAME in zip_archive.namelist()
+        assert PIXELSET_EXPORT_PIXELS_FILENAME in zip_archive.namelist()
+
+    def test_export_pixels_without_pixelsets(self):
```

Comment:
You don't' need to prefix your test names with `test_export_pixels` since your class name is already a kind of namespace.

Reply:
So I would have named this test: `test_export_empty_pixelset`. WDYT?

Reply:
ok